members/views/auth/by_committee.py

```diff
@@ -0,0 +1,9 @@
+from django.http import HttpResponse
+from members.models import Committee
+
+
+def webcie(request):
+    committee = Committee.objects.filter(name__exact="Web Committee").first()
```

Comment:
What does this do when the committee doesn't exist?
Should there just be a none-check  and a 404 if committee doesn't exist?